tsfc/logging.py

```diff
@@ -0,0 +1,11 @@
+"""Logging for TSFC."""
+
+from __future__ import absolute_import
+
+import logging
+
+logger = logging.getLogger('tsfc')
+logger.setLevel(logging.INFO)
+
+default_handler = logging.StreamHandler()
```

Comment:
A warning is printed if no handler is registered.

Firedrake will probably want to remove this handler, and register an "`MPISteamHandler`". However, I also don't TSFC to rely on Firedrake doing some stuff for it.